gem/styles/gem-yegna/modules/_m-article-teaser-stacklist.scss

```diff
@@ -85,13 +85,15 @@
     #{$block}__images {
       display: block;
       @media screen and (min-width: 62.000em)  {
-        //width: 750px;
         max-height: 422px;
       }
+      @media screen and (min-width: 72.188em) {
+        width: 785px;
+      }
     }
     #{$block}__thumbnail {
       @media screen and (min-width: 62.000em)  {
-        min-width: 750px;
+        min-width: 785px;
```

Comment:
is this min value used in other templates? 
Might be worth making a variable

Reply:
Thanks Sewa. It's not used elsewhere or even in a consistent manner atm so no.